minix/lib/libacpi/acpi.c

```diff
@@ -0,0 +1,95 @@
+#define _SYSTEM
```

Comment:
Not necessary but a better way of doing this is to include <minix/drivers.h>, which already defines and includes a bunch of standard stuff for services.